packages/wonder-blocks-core/util/add-style.js

```diff
@@ -20,23 +20,24 @@ export default function addStyle<T: React.AbstractComponent<any> | string>(
         style: StyleType,
         ...
     }) {
-        const {style, ...tmpOtherProps} = props;
+        const {className, style, ...tmpOtherProps} = props;
         // NOTE(jeresig): We need to cast the remaining props to be the right
         // value to ensure that they're typed properly.
         const otherProps: React.ElementConfig<T> = (tmpOtherProps: any);
         const reset =
             typeof Component === "string" ? overrides[Component] : null;
 
-        const {className, style: inlineStyles} = processStyleList([
-            reset,
-            defaultStyle,
-            style,
-        ]);
+        const {
+            className: aphroditeClassName,
+            style: inlineStyles,
+        } = processStyleList([reset, defaultStyle, style]);
 
         return (
             <Component
                 {...otherProps}
-                className={className}
+                className={[aphroditeClassName, className]
```

Comment:
How precedence would work in this case? I mean, what prop would get more importance when rendering? `style` or `className`?

I'd like to understand how the result would be impacted if the user sets both props.

e.g. what happens in this case?

```
style.inlineStyle = { color: "red"};
.cssDefinedStyle {
  color: blue;
}

<Link style={styles.inlineStyle} className="cssDefinedStyle" />
```

Reply:
Precedence for CSS classes is based on the order the classes are defined in, not the order they appear in the `className`.